gce/dns-records.tf

```diff
@@ -11,7 +11,7 @@ resource "google_dns_record_set" "wildcard" {
   name = "*.${var.env}-hipache.${var.dns_zone_name}"
   type = "CNAME"
   ttl = "60"
-  rrdatas = ["${google_dns_record_set.router.name}"]
+  rrdatas = ["${google_compute_forwarding_rule.router_https.ip_address}"]
```

Comment:
This is the IP of the LB GCE provides (they give you the IP directly instead of CNAME - unlike AWS). In short, this is same target as what <env>-hipache record points to.